ftdetect/hocon.vim

```diff
@@ -0,0 +1 @@
+au BufRead,BufNewFile *.conf set filetype=hocon
```

Comment:
please don't add this on such a generic file ending. For the test to pass, just add it to scripts/minimal_init.lua

Reply:
Ok. Will do tomorrow. Thanks

Reply:
done. Now tests should pass